docs/modules/spawn.mdx

```diff
@@ -2,8 +2,88 @@
 title: Spawn
 ---
 
-:::note
+:::warning
 This addon requires you to have a matching version of EssentialsX installed.
 :::
 
-## Frequently Asked Questions
\ No newline at end of file
+:::note
+[Vault](https://github.com/milkbowl/Vault) is required for any group related functionalities documented below.
+:::
+
+EssentialsX Spawn lets you control where players spawn both when they first join the server and when they die.
+You can use the in-game `/setspawn` command to set a spawn location for everyone on your server as well as for specific groups.
+
+## Configuration
+You can access the EssentialsSpawn configuration by opening `/plugins/Essentials/config.yml`, and scrolling to the bottom of the file.
```

Comment:
This should say `and navigate to the section labeled "EssentialsX Spawn"`. I would like to avoid suggesting scrolling to a position to try and avoid confusion as to where it may start/stop, and being consistent across the rest of the docs.